src/Image_Generators/Batch_Sorter.py

```diff
@@ -1,7 +1,6 @@
 import bpy
 import os
 import sys
-import timeit
```

Comment:
I could be mistaken but I think `timeit` is used to print something along the lines of "generated in x seconds" in the console